template-sample-app/template/package.dev.json

```diff
@@ -11,12 +11,12 @@
     "@material-ui/core": "^4.11.0",
     "@material-ui/icons": "^4.9.1",
     "@material-ui/lab": "^4.0.0-alpha.56",
-    "@reduxjs/toolkit": "^1.1.0",
+    "@reduxjs/toolkit": "^1.5.0",
     "@testing-library/jest-dom": "^4.2.4",
     "@testing-library/react": "^9.3.2",
     "@testing-library/user-event": "^7.1.2",
     "d3-scale": "^3.2.3",
-    "deck.gl": "8.3.8",
+    "deck.gl": "8.4.0-alpha.4",
```

Comment:
This change has to be applied in more places:

![image](https://user-images.githubusercontent.com/458196/104479744-13425f80-55c4-11eb-9f8d-fedd5c726739.png)

Reply:
... and same for reduxjs/toolkit